python_modules/dagster/dagster_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -5284,6 +5284,8 @@ def test_default_concurrency(
         if not self.can_set_concurrency_defaults():
             pytest.skip("storage does not support setting global op concurrency defaults")
 
+        assert not storage.initialize_concurrency_limit_to_default("foo")
```

Comment:
what exactly does returning False here indicate, should this be raising an exception instead on failure? I'm not seeing anywhere outside of tests that we actually use the return value here

Reply:
ah ok it looks like before https://github.com/dagster-io/dagster/pull/27112/files we were changing behavior based on the return value, now we are not

This seems very easy to mess up in the future / innocuously change to make it always return True or something, probably worth a big comment in both the test and in the implementation itself